packages/images-plugin/lib/image-element.js

```diff
@@ -88,10 +86,17 @@ export default class ImageElement {
   }
 
   /**
-   * @returns {?string} - the base64-encoded image which has matched marks
+   * @returns {string} - the base64-encoded original image used for matching
+   */
+  get originalImage() {
+    return this.template.toString('base64');
+  }
+
+  /**
+   * @returns {string|null} - the base64-encoded image which has matched marks
    */
   get matchedImage() {
-    return this.b64MatchedImage;
+    return this.match?.toString('base64') ?? null;
```

Comment:
doesn't `this.match?.toString('base64')` already return null if `this.match` is null?

Reply:
👍